cmd/data-scanner.go

```diff
@@ -1245,8 +1236,25 @@ func applyExpiryOnNonTransitionedObjects(ctx context.Context, objLayer ObjectLay
 	if lcEvent.Action.DeleteAll() {
 		opts.DeletePrefix = true
 	}
+	var (
+		dobj ObjectInfo
+		err  error
+	)
+	defer func() {
+		if err != nil {
+			return
+		}
```

Comment:
Do we count failures as well?

Reply:
These metrics are meant to count only successful ILM actions.

Reply:
I am saying why not count errors as well?